packages/chat-client/src/controllers/engine.ts

```diff
@@ -692,7 +719,13 @@ export class ChatEngine extends IChatEngine {
 
       if (!decodedPayload) throw new Error("Empty ID Auth payload");
 
-      const { inviteKeyPub } = this.client.chatKeys.get(
+      if (decodedPayload.act !== "invite_proposal") {
+        this.client.logger.error(
```

Comment:
This is only logging but not throwing/aborting the flow at this stage, presumably this is a terminating condition/exception?